packages/grid/theme/lumo/vaadin-grid-styles.js

```diff
@@ -174,7 +174,7 @@ registerStyles(
       background: var(--lumo-primary-color-50pct);
     }
 
-    #scroller:not([ios]) [part~='row'][dragstart]:not([dragstart=''])::after {
```

Comment:
See https://github.com/vaadin/vaadin-grid/pull/1680#discussion_r351345102 There's no longer a reason to disable the row count indicator on iOS generally